multicluster/hack/generate-manifest.sh

```diff
@@ -118,6 +118,8 @@ then
     mkdir config && cd config
     cp $KUSTOMIZATION_DIR/overlays/leader-ns/prefix_transformer.yaml .
     sed -ie "s/antrea-multicluster/$NAMESPACE/g" prefix_transformer.yaml
+    cp $KUSTOMIZATION_DIR/overlays/leader-ns/webhook_patch.yaml .
+    sed -ie "s/antrea-multicluster/leader/g" webhook_patch.yaml
```

Comment:
I don't get this, I suppose you can set it as leader in the manifest directly?

Reply:
removed.